src/+datools/+statistical/+ensemble/EnF.m

```diff
@@ -75,6 +79,15 @@
         end
 
         function forecast(obj)
+            % Description:
+            % Method to propagate the model forward in time and update the
+            % ensembles accordingly
+            %
+            % INPUT PARAMETERS:
```

Comment:
I don't see any "Input parameters" in the standard.

Reply:
He told me he wanted the input and output parameters for each methods and what the method does in general.

Reply:
Then do it in the matlab style

see this page https://www.mathworks.com/help/matlab/matlab_prog/add-help-for-your-program.html

that will have you do input and output similar to the official matlab docs